### PR TITLE
twa: 1.8.0 -> 1.9.1

### PR DESCRIPTION
**Pull request by Avaq**
###### Motivation for this change

To close #72689 by providing a working alternative

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Avaq** [comment]
I have found two minor issues in this version of `twa`: https://github.com/trailofbits/twa/issues/66 and https://github.com/trailofbits/twa/issues/67. I'm expecting the author to fix these soon. If this PR has not been merged by that time, I will update the package version and PR title.

**Comment by woodruffw** [comment]
Upstream here: Thanks. I've cut 1.9.1 with both fixes.

**Comment by Avaq** [comment]
Updated to 1.9.1 and retested. Upstream fixes confirmed.

File: pkgs/tools/networking/twa/default.nix [hunk 1]
```diff
@@ -12,18 +13,24 @@
...
 
   dontBuild = true;
 
-  buildInputs = [ makeWrapper bash gawk curl netcat host.dnsutils ];
+  buildInputs = [ bash
+                  curl
+                  gawk
+                  host.dnsutils
+                  jq
+                  makeWrapper
```
Comment (c0bw3b) [inline pkgs/tools/networking/twa/default.nix]: `makeWrapper` belongs in `nativeBuildInputs` as it's used during build but not necessary at runtime.

**Comment by Avaq** [comment]
@c0bw3b I moved `makeWrapper` to the nativeBuildInputs, also describing the change in the commit message.

**Review by c0bw3b — APPROVED** [review]
Builds and runs

**Pull request merged**